bin/bootstrax

```diff
@@ -1340,6 +1345,25 @@ def run_strax(run_id, input_dir, targets, readout_threads, compressor,
             f.write(exc_info)
         raise
 
+def last_file_write_time(match_location: str) -> datetime:
+    """
+    Get the datetime in UTC of the last file that was written in raw-records*
+    :param match_location: Location where runs are written to
+    :return: TZ aware timestamp
+    """
+    last_time = datetime.fromtimestamp(0).replace(tzinfo=pytz.utc)
+    matched_folders = glob(match_location)
+    for folder in matched_folders:
+        # check last three files only
+        chunk_files = sorted(glob(os.path.join(folder, '*')))[-3:]
+        for chunk in chunk_files:
+            # Check that we did not rename this file since the glob above
+            if os.path.exists(chunk):
```

Comment:
Do we care about the edge case where _all_ the files we check were renamed, so that we end up returning the dawn of time, or is that sufficiently unlikely?

Reply:
I think the `os.path.exists(chunk)`  check is already quite conservative - I doubt it every actually returns False, the edge case where _all_ things are renamed is somethings I consider sufficiently unlikely. It's an easy fix if my imagination is ever proven to be too limited 😉